<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainMenu</class>
 <widget class="QWidget" name="MainMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>244</width>
    <height>372</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainMenu</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>20</y>
     <width>81</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>Balance</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>100</y>
     <width>81</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>Withdraw</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>260</y>
     <width>81</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>Mini Statement</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>180</y>
     <width>81</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>Pin Change</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>141</width>
     <height>321</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
